src/main/webapp/receipt-analysis.html

```diff
@@ -23,6 +23,10 @@
 <title>Receipt Analysis</title>
 
 <body onload="load()">
+  <button id="home-button" href="/" class="btn btn-info btn-sm mt-2 ml-2" type="button" onclick="redirectHome()">
```

Comment:
There is a redirectHome() in common.js and receipt-analysis.js. They're both loaded for this page. I feel that could cause some confusion, maybe name them differently.